### ISSUE TITLE
Inconsistend Date Formats

### ISSUE THREAD

**Issue by xoxoedg** [M-Media-Group/Covid-19-API#35]
Hi,

i found that the Date Formats on Vaccines and Cases Endpoint are different.

Vaccines:
**"2021/12/25 00:00:00+00"**

Cases:
**"2021-12-25 04:21:37"**

I dont know if this is relevant for you, but just wanted to let you knows.

e.g. (Vaccines)
![image](https://user-images.githubusercontent.com/84626740/147393307-a80a1a15-3baa-4da9-b81b-31155a3faed8.png)

e.g.(Cases)
![image](https://user-images.githubusercontent.com/84626740/147393311-72e26951-106b-4ade-b4dc-3188a7ac9fc8.png)

**Comment by mwargan** [comment]
Good catch, I'll probably standardise this in the near future, but it could have large knock-on effects to people already using this API and used to these date formats (so t those, subscribe to notifications to this issue for updates).

**Comment by xoxoedg** [comment]
Hi do you have any news regarding those changes
